src/Controllers/HomeController.php

```diff
@@ -19,10 +21,22 @@ public function showIndex()
     {
         $components = Component::orderBy('order')->orderBy('created_at')->get();
 
+        $allIncidents = [];
+
+        foreach (range(1, 8) as $i) {
+            $date = Carbon::now()->subDays($i);
```

Comment:
What I mean is that I'm getting a "Undefined variable: incidentDate" so either change the value of $incidentDate to $date or change the $incidentDate below to $date :)